src/patterns/dosdp.yaml

```diff
@@ -0,0 +1,28 @@
+name: Update DOSDP pattern files
```

Comment:
move this file to `.github/workflows` in the top level of the repository, see https://github.com/obophenotype/bio-attribute-ontology/blob/master/.github/workflows/dosdp.yml

Reply:
Moved. But I just noticed, it is supposed to be '.yml' and not '.yaml'?

Reply:
It does not
Matter but yml is a bit more popular. I would standardise on this